torch/csrc/jit/codegen/cuda/inlining.cpp

```diff
@@ -11,12 +11,18 @@ namespace fuser {
 namespace cuda {
 
 MaxPosCalculator::MaxPosCalculator(
-    const std::unordered_set<IterDomain*>& uninlinable_ids)
+    const std::unordered_set<IterDomain*>& uninlinable_ids,
+    bool compute_at_only)
     : uninlinable_ids_(uninlinable_ids) {
-  buildUnmappableDims();
+  buildUnmappableDims(compute_at_only);
```

Comment:
nit: Why not just
```
  // When used for computeAt only, i.e., without inlining storeAt
  // positions, the restriction below does not apply
  if (compute_at_only) {
    buildUnmappableDims();
  }
```

Reply:
That should work too, but I also thought `buildUnmappableDims` could populate some other IDs that should not be mapped, so I didn't want to completely skip calling `buildUnmappableDims()`. Not something I have a strong preference.

Reply:
That's a good point. It might probably be still needed if we want to fix https://github.com/csarofeen/pytorch/issues/2017, I will think more about this, but for this PR, I am OK with leaving it as is.